nucliadb_writer/nucliadb_writer/tus/gcs.py

```diff
@@ -194,7 +194,8 @@ async def start(self, dm: FileDataMangaer, path: str, kbid: str):
             filename = "file"
         else:
             filename = dm.filename
-        metadata = json.dumps({"NAME": filename})
+        metadata = json.dumps({"metadata": {"FILENAME": filename}})
```

Comment:
Following google documentation: https://cloud.google.com/storage/docs/json_api/v1/objects#resource-representations

Custom metadata needs to be nested under `metadata` key. 
As of the `name` key, we don't really need it, as it is already being set at the `init_url` on line 189

Reply:
Great catch!